platforms/substrate/charts/dscp-ipfs-node/templates/statefulset.yaml

```diff
@@ -167,7 +169,7 @@ spec:
                 jq -r 'if .errors then . else .auth.client_token end')
 
               validateVaultResponse 'vault login token' "${VAULT_CLIENT_TOKEN}"
-              vault_secret_key="${CERTS_SECRET_PREFIX}/ipfs"
+              vault_secret_key="${VAULT_SECRET_ENGINE}"/"${VAULT_SECRET_PREFIX}/ipfs"
```

Comment:
Why is this not using bevel-scripts?

Reply:
Already checked the script it is not working, That is why we are having code here it self